subworkflows/local/utils_nfcore_rnafusion_pipeline/main.nf

```diff
@@ -129,7 +130,7 @@ workflow PIPELINE_COMPLETION {
                 plaintext_email,
                 outdir,
                 monochrome_logs,
-                multiqc_report.toList()
+                multiqc_reports.getVal(),
```

Comment:
Should the final "," in `multiqc_reports.getVal(),` be removed?

Reply:
Removed